controller/tap/server.go

```diff
@@ -426,18 +432,53 @@ func (s *server) translateEvent(orig *proxy.TapEvent) *public.TapEvent {
 	return ev
 }
 
+func newHTTPServer(k8sAPI *k8s.API, controllerNamespace string) *http.Server {
+	var (
+		aggregationLayerConfigMapName = "extension-apiserver-authentication"
+	)
+
+	cm, err := k8sAPI.Client.CoreV1().
```

Comment:
I tried to include that but it didn't seem to work. I think that its because k8sAPI is not a `kubernetes.Interface` which is an interface we create. It looks like this file only has `*k8s.API` available in `NewServer`.

Reply:
Ah my mistake! This is an `API` from `controller/k8s`, I thought it was a `KubernetesAPI` from `pkg/k8s`. Perhaps this is a hint to myself that we could consolidate these at some point.